backend/tests/Feature/Supervisor/WorkOrderMachinesTest.php

```
@@ -0,0 +1,155 @@
+use App\Models\Line;
+use App\Models\User;
+use App\Models\WorkOrder;
+use App\Models\Workstation;
+use App\Models\WorkstationState;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Inertia\Testing\AssertableInertia;
+use Tests\TestCase;
+
+class WorkOrderMachinesTest extends TestCase
+{
+    use RefreshDatabase;
```

PHPStan (2.2.2)

[error] Class Tests\Feature\Supervisor\WorkOrderMachinesTest uses unknown trait Illuminate\Foundation\Testing\RefreshDatabase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(trait.notFound)